### ISSUE TITLE
Meatabolism

### ISSUE THREAD

**Issue by drc007** [OpenSourceMalaria/OSM_To_Do_List#334]
I thought it might be useful to have a separate thread to discuss metabolism.

The program I used predicts the potential sites of metabolism but does not predict rates so I can't say with any reliability which is the most likely sites for metabolism. But I suspect cleavage of the ether linkage is likely to be significant.

Apologies if this is obvious, but the product of cleavage of the ether side chain would leave an oxygen on the triazolopyridine ring. The image on the slide suggests the cleavage occurs between the oxygen and the ring.
![dealkylation](https://cloud.githubusercontent.com/assets/7080516/8900515/a522aa3a-3438-11e5-95f3-61331bc74862.png)

Have you tried treating one of the structures with mCPBA to see if any major oxygenated species are generated?

**Comment by mattodd** [comment]
Hi Chris - great to start a thread on this. Solving metabolic liabilities is the biggest and most important task we have. It may be we need to split this into composite to do items - Github works best when it's used as a place to post specific queries/items that can be neatly closed - but let's keep this for now to get us all going. Teaching today prevents me immediately posting the meeting minutes from yesterday, but I'll get to that this week hopefully.

The fragmentation is as you suggest in the [original scheme](http://malaria.ourexperiment.org/biological_data/8225/Summary_of_Series_4_Metabolic_Data__July_2015.html). The line you're referring to is indicating where the extended aromatic system is being oxidised. So yes the hydroxy-triazolopyrazine compound you draw above is the one that I think David Shackleford is wanting to test.

Not tried any series 4 compounds with mCPBA AFAIK. Is that really a good mimic? I have no doubt we can produce something oxidised with some conditions, but we'd need relevant conditions, surely?

**Comment by drc007** [comment]
I was not suggesting mCPBA as a mimic, merely as a way to generate authentic samples that Dave Shackleford could use to confirm (or exclude) the identity of metabolites. Don't underestimate the challenge though. Can require considerable effort and technical expertise to isolate and unambiguously identify small amounts of what can be very polar compounds.

**Comment by mattodd** [comment]
Ah. OK, well we can probably make directly from the chloride in bulk if needs be. We actually [nearly made the molecule by accident, in OSM-S-280](http://malaria.ourexperiment.org/biological_data/11080/Results_for_samples_sent_for_in_vitro_efficacy_evaluation_against_Plasmodium_falciparum_asexual_blood_stage.html).

**Comment by drc007** [comment]
I just noticed you made the N-oxide MMV668822, have you looked at this as a potential metabolite of MMV675960?

**Comment by mattodd** [comment]
No we haven't. The N-oxide was an inherited datapoint. What's the
experiment you're thinking of - seeing if 960 produces 822 in vitro?

On 28 July 2015 at 16:52, Chris Swain <notifications@github.com> wrote:

>  I just noticed you made the N-oxide MMV668822, have you looked at this
> as a potential metabolite of MMV675960?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/334#issuecomment-125472042>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/people/matthew.todd.php
W http://opensourcemalaria.org/ | W http://opensourcetb.org/ | W
http://opensourcepharma.net/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by drc007** [comment]
Exactly

**Comment by mattodd** [comment]
...but further that we'd then learn whether we ought to be blocking such a
transformation? We know we can't replace the N with a C-H, and that we
can't add a methyl group to the adjacent position (MMV669541).

On 28 July 2015 at 23:01, Chris Swain <notifications@github.com> wrote:

>  Exactly
>
> Sent from Chris's iPhone
>
> > On 28 Jul 2015, at 13:59, Mat Todd <notifications@github.com> wrote:
> >
> > No we haven't. The N-oxide was an inherited datapoint. What's the
> > experiment you're thinking of - seeing if 960 produces 822 in vitro?
> >
> > On 28 July 2015 at 16:52, Chris Swain <notifications@github.com> wrote:
> >
> > > I just noticed you made the N-oxide MMV668822, have you looked at this
> > > as a potential metabolite of MMV675960?
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/334#issuecomment-125472042
> >
> > > .
> > >
> >
> >
> >
> > --
> > MATTHEW TODD | Associate Professor
> > School of Chemistry | Faculty of Science
> >
> > THE UNIVERSITY OF SYDNEY
> > Rm 519, F11 | The University of Sydney | NSW | 2006
> > T +61 2 9351 2180 | F +61 2 9351 3329 | M +61 415 274104
> > E matthew.todd@sydney.edu.au | W
> > http://sydney.edu.au/science/people/matthew.todd.php
> > W http://opensourcemalaria.org/ | W http://opensourcetb.org/ | W
> > http://opensourcepharma.net/
> >
> > CRICOS 00026A
> > This email plus any attachments to it are confidential. Any unauthorised
> > use is strictly prohibited. If you receive this email in error, please
> > delete it and any attachments.
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/334#issuecomment-125598162>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/people/matthew.todd.php
W http://opensourcemalaria.org/ | W http://opensourcetb.org/ | W
http://opensourcepharma.net/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by drc007** [comment]
Yes, you would also know if this was a major route. If it was it might be worth looking at changing the bicycling ring more extensively.

**Comment by MFernflower** [comment]
Provided it doesn't shred apart the molecule too vigorously, could ozone be a possible simulation of biological oxidation?
